pkg/volume/cinder/cinder_util.go

```diff
@@ -139,7 +141,26 @@ func (util *CinderDiskUtil) CreateVolume(c *cinderVolumeProvisioner) (volumeID s
 	// Cinder works with gigabytes, convert to GiB with rounding up
 	volSizeGB := int(volume.RoundUpSize(volSizeBytes, 1024*1024*1024))
 	name := volume.GenerateVolumeName(c.options.ClusterName, c.options.PVName, 255) // Cinder volume name can have up to 255 characters
-	name, err = cloud.CreateVolume(name, volSizeGB, c.options.CloudTags)
+	vtype := ""
```

Comment:
Cinder volumetype is not preset. You can set anything or nothing there. As long as there is a match volumetype in Cinder, the API returns a success; otherwise the API gets an error telling no matching volumetype is found.

If volumetype is not provided in the API call, it is set to "" in the API implementation later.